x-pack/plugins/data_visualizer/public/application/common/components/expanded_row/geo_point_content_with_map/geo_point_content_with_map.tsx

```diff
@@ -29,6 +29,8 @@ export const GeoPointContentWithMap: FC<{
     services: { maps: mapsPlugin, data },
   } = useDataVisualizerKibana();
 
+  const uniqueExamples = useMemo(() => getUniqCoordinates(stats?.examples), [stats?.examples]);
```

Comment:
Do the examples come from `getFieldExamples` in https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/models/data_visualizer/data_visualizer.ts? Just curious as to whether the check for uniqueness should be moved to the server to ensure we stand a good chance of getting the maximum number of examples (10?) displayed in the UI.

Reply:
If it's from `getFieldExamples` we should probably add a specific case for the `GEO_POINT` and `GEO_SHAPE`, rather than it falling through to the `default` case.

Reply:
Updated [here](https://github.com/elastic/kibana/pull/136236/commits/9c6ca41f28f44b08d0e2f5901009824352a952f2)